### ISSUE TITLE
FR: Put Info an hidden Items to the menu, not the palette

### ISSUE THREAD

**Issue by chrisgrieser** [AlexBieg/obsidian-better-command-palette#53]
While I like the new feature of hiding certain commands, I do think that both the list of hidden items as well as the `x` to hide them do clutter up the palette quite a bit. I think putting options to remove commands as well as an overview of hidden commands into the settings menu would be better, since I'd expect it to be there anyway?

**Comment by AlexBieg** [comment]
@chrisgrieser Yeah, I agree to a certain extent. There were a few considerations that led to this approach, but I definitely think there is room for improvement.

1. Regarding the `X`s: I initially wanted these to be only shown on hover. Unfortunately, this was not a UI/UX that works in a mobile environment. A possible compromise here would be to allow you to hide the `X` via a setting.
2. Removing the commands on the settings page: I had considered this as well but concluded that 99% of the time, when you want to remove a command, it's while you are actively using the palette. Including that functionality right there and then is (at least for me) a much more usable experience. While this doesn't preclude adding a way to do it in the settings, I would like to avoid duplication of functionality. The more I duplicate stuff like that, the harder it becomes to add/fix things in the future.
3. Showing a list of your hidden commands in settings: Honestly went back and forth on this one while initially developing this feature. I decided that it wasn't part of the MVP, but I am open to adding something. For context, I didn't think people would need it because the only time I need to know if a command is hidden is when I am trying to use it. If I'm trying to use it, meaning I have already typed something into the palette that would search for that command, then it will show up in that hidden command notice.
4. The notice telling you how many commands are currently hidden: I am very reticent to hide this in any circumstances. I understand it can add a bit of clutter to the screen, but it is the only feedback that you have something hidden. I think sacrificing a bit of cleanliness for useful actionable information is worth it in this case.

Feel free to push back, but these are the action items I see from this. Let me know your thoughts 😄 

Possible Action Items
1. Add setting to remove/hide `X`s in the palette
2. Add setting to view all currently hidden commands
